### DISCUSSION TITLE
TeslaMate or Grafana hangs raspberry pi

### DISCUSSION THREAD

**Discussion by douwevt** [teslamate-org/teslamate#4283] [General]
Since one of the last updates, when starting TeslaMate on my Raspberry Pi, after very little time, the system gets very very slow. Pressing an enter key will take minutes of not hours to respond. Debugging is therefore also not possible. Anyone has seen this kind of behavior and was able to solve it?

**Comment by cwanja** [comment]
TeslaMate has not updated in three months. Has it been happening for three months?

Are you running on a microSD card? It could be that your SD card is failing.

Is anything else running on the Pi? If so, does it run slowly as well?

I'd make sure you have backups if it continues.

**Reply by JakobLichterfeld** [reply]
That, or he is also affected by #4225 which is fixed by #4228 which will be releases in near future

**Reply by douwevt** [reply]
Both are good comments, I am not running anything else on the Pi, but I will try to run an fsck on it. I am not sure exactly when it went wrong, I don't look on it too often, but usually at least once a month, so I thought it might have to do with the combination of some Linux update and the latest update on TeslaMate or so. I will keep an eye on that #4228, but installing it might be a challenge with the current response times I get on the system. ;-) (so will the fsck be a challenge I suppose)

**Reply by douwevt** [reply]
fsck on the drive gave no errors, so I will wait for that #4228 to come along. Thanks everyone
